app/Api/ProjectApi.php

```diff
@@ -62,26 +67,28 @@ public function getProjectActivity($project_id)
         return $this->helper->projectActivity->getProjectEvents($project_id);
     }
 
-    public function createProject($name, $description = null)
+    public function createProject($name, $description = null, $owner_id=0, $identifier=null)
     {
         $values = array(
             'name' => $name,
-            'description' => $description
+            'description' => $description,
+            'identifier' => $identifier
         );
 
         list($valid, ) = $this->projectValidator->validateCreation($values);
-        return $valid ? $this->projectModel->create($values) : false;
+        return $valid ? $this->projectModel->create($values, $owner_id) : false;
     }
 
-    public function updateProject($id, $name, $description = null)
+    public function updateProject($id, $name, $description = null, $owner_id=0,  $identifier=null)
     {
         $values = array(
             'id' => $id,
             'name' => $name,
-            'description' => $description
+            'description' => $description,
+            'identifier' => $identifier
         );
 
         list($valid, ) = $this->projectValidator->validateModification($values);
-        return $valid && $this->projectModel->update($values);
+        return $valid && $this->projectModel->update($values, $owner_id);
```

Comment:
The method `ProjectModel::update()` do not have the argument `owner_id`. See https://github.com/fguillot/kanboard/blob/master/app/Model/ProjectModel.php#L429